src/In.ProjectEKA.HipService/Startup.cs

```diff
@@ -60,6 +63,7 @@ public Startup(IConfiguration configuration)
                 .AddDbContext<ConsentContext>(options =>
                     options.UseNpgsql(Configuration.GetConnectionString("DefaultConnection"),
                         x => x.MigrationsAssembly("In.ProjectEKA.HipService")))
+                .AddHangfire(config => { config.UseMemoryStorage(); })
```

Comment:
This should be coming from different project, so that it will be easy for anyone to replace the data storage.